lib/suppressions.cpp

```diff
@@ -448,19 +448,19 @@ const std::list<Suppressions::Suppression> &Suppressions::getSuppressions() cons
 }
 
 /**
- * This function require that both tokens and suppressions are sorted in line
- * number order.
+ * Loop on mSuppressions over and oer again might not be so fast.
```

Comment:
The placement here indicates that this comment will describe what the function is doing. But the text does not say that.

It feels more like some TODO comment to speedup.. 

I don't feel very worried that this function will be a bottleneck since it's only called once on each file configuration right?